# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/WSRunObject.h
# Commit: ced5b6a8ab8d
# Full Hash: ced5b6a8ab8da9e2401026d6dc0baeb4a16dfc74
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-03-02 21:27:32
# Regressor Bug: 1618089
# File Overlap Count: 2
# Description:
#   Bug 1618089 - part 3: Rename `WSRunScanner::Get(Previous|Next)CharPoint(const WSPoint&)` to `WSRunScanner::Get(Previous|Next)CharPointFromPointInText()` r=m_kato
#   
#   For making it easier to review the following patches, rename the overloads
#   of `WSRunScanner::Get(Previous|Next)CharPoint()` which take `WSPoint`.
#   
# ==============================================================================

diff -r 54a21814751f -r ced5b6a8ab8d editor/libeditor/WSRunObject.h
--- a/editor/libeditor/WSRunObject.h	Mon Mar 02 07:33:56 2020 +0000
+++ b/editor/libeditor/WSRunObject.h	Mon Mar 02 07:35:34 2020 +0000
@@ -523,12 +523,13 @@
                              bool aForward) const;
 
   /**
-   * GetNextCharPoint() returns next character's point of aPoint.  If there is
-   * no character after aPoint, mTextNode is set to nullptr.
+   * GetNextCharPoint() and GetNextCharPointFromPointInText() return next
+   * character's point of aPoint.  If there is no character after aPoint,
+   * mTextNode is set to nullptr.
    */
   template <typename PT, typename CT>
   WSPoint GetNextCharPoint(const EditorDOMPointBase<PT, CT>& aPoint) const;
-  WSPoint GetNextCharPoint(const WSPoint& aPoint) const;
+  WSPoint GetNextCharPointFromPointInText(const WSPoint& aPoint) const;
 
   /**
    * LookForNextCharPointWithinAllTextNodes() and
@@ -536,8 +537,8 @@
    * GetNextCharPoint(const EditorRawDOMPoint&) and GetPreviousCharPoint(const
    * EditorRawDOMPoint&).  When the container isn't in mNodeArray, they call one
    * of these methods.  Then, these methods look for nearest text node in
-   * mNodeArray from aPoint. Then, will call GetNextCharPoint(const WSPoint&) or
-   * GetPreviousCharPoint(const WSPoint&) and returns its result.
+   * mNodeArray from aPoint. Then, will call GetNextCharPointFromPointInText()
+   * or GetPreviousCharPointFromPointInText() and returns its result.
    */
   template <typename PT, typename CT>
   WSPoint LookForNextCharPointWithinAllTextNodes(
@@ -569,12 +570,13 @@
                             nsINode* aBlockParent) const;
 
   /**
-   * GetPreviousCharPoint() returns previous character's point of of aPoint.
-   * If there is no character before aPoint, mTextNode is set to nullptr.
+   * GetPreviousCharPoint() and GetPreviousCharPointFromPointInText() return
+   * previous character's point of of aPoint. If there is no character before
+   * aPoint, mTextNode is set to nullptr.
    */
   template <typename PT, typename CT>
   WSPoint GetPreviousCharPoint(const EditorDOMPointBase<PT, CT>& aPoint) const;
-  WSPoint GetPreviousCharPoint(const WSPoint& aPoint) const;
+  WSPoint GetPreviousCharPointFromPointInText(const WSPoint& aPoint) const;
 
   char16_t GetCharAt(dom::Text* aTextNode, int32_t aOffset) const;
 
